### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by OybekKayumov**
## Hello, in this PR:

 - Create a new app. 
 - Initialize the project with Git.
 - Project has a Postgres database set up.

 - Build project schema.
 - Create and run the necessary migration files.
 - Table and column names match the ERD diagram.
 - Foreign keys included.

### CODE REVIEW

File: db/migrate/20220706064618_create_post.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+
+      t.timestamps
```
Comment (keroloussamy) [inline db/migrate/20220706064618_create_post.rb]: - [ ] Kindly in the Post table you have to add the `comments_counter` and the `likes_counter` as integers. To follow the ERD.

File: db/migrate/20220706065102_add_user_ref_to_post.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :users, null: false, foreign_key: true
```
Comment (keroloussamy) [inline db/migrate/20220706065102_add_user_ref_to_post.rb]: - [ ] Kindly in this way the foreign key will be `user_id`, to make it `author_id` update your migration to `add_reference :posts, :author, foreign_key: { to_table: 'users' }`. Or you can leave it as it is and just add this line `rename_column :posts, :user_id, :author_id` to update the foreign key name.

File: db/migrate/20220706064953_add_user_ref_to_like.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```
Comment (keroloussamy) [inline db/migrate/20220706064953_add_user_ref_to_like.rb]: - [ ] Kindly in this way the foreign key will be `user_id`, to make it `author_id` update your migration to `add_reference :likes, :author, foreign_key: { to_table: 'users' }`. Or you can leave it as it is and just add this line `rename_column :likes, :user_id, :author_id` to update the foreign key name.

File: db/migrate/20220706064840_add_user_ref_to_comment.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```
Comment (keroloussamy) [inline db/migrate/20220706064840_add_user_ref_to_comment.rb]: - [ ] Kindly in this way the foreign key will be `user_id`, to make it `author_id` update your migration to `add_reference :comments, :author, foreign_key: { to_table: 'users' }`. Or you can leave it as it is and just add this line `rename_column :comments, :user_id, :author_id` to update the foreign key name.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @OybekKayumov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220706064840_add_user_ref_to_comment.rb (hunk 4 above)
Comment (OybekKayumov) [reply db/migrate/20220706064840_add_user_ref_to_comment.rb]: Hello @keroloussamy !
Thank you for your feedback

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @OybekKayumov, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Kudos! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**